archivers/par2/Portfile

```diff
@@ -2,8 +2,8 @@ PortSystem        1.0
 PortGroup         compiler_blacklist_versions 1.0
 
 name              par2
-version           0.4
-revision          4
+version           0.8.1
+revision          1
```

Comment:
the "revision" field should be reset to zero when updating the "version"

Reply:
Done